### DISCUSSION TITLE
hexo g报错，生成的文件全是空的

### DISCUSSION THREAD

**Discussion by icehomegre** [hexojs/hexo#5340] [Question / Answer]
![bandicam 2023-10-31 08-41-28-323](https://github.com/hexojs/hexo/assets/141923612/01b6a6b6-3fd5-48ee-8af6-6937f42a6d99)
上图是报错，这一种错误重复报了很多次。
生成的html文件全部都是空文件，大小0kB
系统为win10 22h2
已安装nodejs最新版
之前用旧的电脑是可以通过的，但是换了新电脑，把文件复制过来，完善了个人配置之后就不能用了。

This problem has been many times.
It's OK on my old computer, but after I copied files to the new computer and filled personal information, it cannot run.

**Comment by yoshinorin** [comment]
@icehomegre 

Perhaps it's caused by theme settings, isn't it? Please check the below issue.

https://github.com/jerryc127/hexo-theme-butterfly/issues/781

Thank you :)

**Reply by icehomegre** [reply]
Thank you. It was found that there was a problem with the package. After installing the search plugin, the problem was resolved.
